dll/win32/browseui/shellbrowser.cpp

```diff
@@ -709,7 +709,10 @@ CShellBrowser::CShellBrowser()
     fCurrentShellViewWindow = NULL;
     fCurrentDirectoryPIDL = NULL;
     fStatusBar = NULL;
-    fStatusBarVisible = true;
+    fStatusBarVisible = SHRegGetBoolUSValueW(L"Software\\Microsoft\\Internet Explorer\\Main",
```

Comment:
What about first run when value is not set ? Default value to be used or registry initialization in ReactOS setup to be modified